packages/pwa-kit-runtime/src/ssr/server/express.js

```diff
@@ -473,11 +477,12 @@ export const getRuntime = () => {
     // Sometimes the runtime APIs are invoked directly as express middlewares.
     // In order to make sure the "this" keyword always have the correct context,
     // we bind every single method to have the context of the object itself
-    for (const property in runtime) {
-        if (runtime[property] instanceof Function) {
-            runtime[property] = runtime[property].bind(runtime)
+    const boundRuntime = {...runtime}
+    for (const property of Object.keys(boundRuntime)) {
```

Comment:
👍  TIL https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#difference_between_for...of_and_for...in